modules/programs/rofi.nix

```diff
@@ -420,7 +429,8 @@ in {
     home.packages = let
       rofiWithPlugins = cfg.package.override
         (old: rec { plugins = (old.plugins or [ ]) ++ cfg.plugins; });
-      rofiPackage = if builtins.hasAttr "override" cfg.package then
+      rofiPackage = if builtins.hasAttr "override" cfg.package
+      && cfg.package.override.__functionArgs ? plugins then
```

Comment:
Ditto about the `hasAttr "override"`.

Reply:
Same case as [here](https://github.com/nix-community/home-manager/pull/2197#discussion_r672913935)

Reply:
Can you replace the `__functionArgs` with `lib.functionArgs`?